### ISSUE TITLE
[DOC]  Modify access control documentation for cross-cluster search 

### ISSUE THREAD

**Issue by gbbafna** [opensearch-project/documentation-website#867]
## What do you want to do?
 
- [x] Request a change to existing documentation

## Tell us about your request. 

It is mentioned in [documentation](https://opensearch.org/docs/latest/security-plugin/access-control/cross-cluster-search/) that 

```
To query indices on remote clusters, users need to have the following permissions for the index, in addition to READ or SEARCH permissions:

indices:admin/shards/search_shards

```

This is not true . `indices:admin/shards/search_shards` is needed only when user tries to do cross cluster search with with query parameter `ccs_minimize_roundtrips=false` . Otherwise , the presence/absence of this permission doesn't hamper the cross cluster search . The default value of `ccs_minimize_roundtrips` is true. 

 
## Do you have any related resources to help us get started? 

Follow the steps mentioned here : https://opensearch.org/docs/latest/security-plugin/access-control/cross-cluster-search/

Without `indices:admin/shards/search_shards` permission , cross cluster searches go through .

Now when we try cross cluster searche with `ccs_minimize_roundtrips=false` , the search fails saying the need for `indices:admin/shards/search_shards` permission

```
GET remote-cluster:books/_search?ccs_minimize_roundtrips=false
```

This would fail with

```
{
  "error" : {
    "root_cause" : [
      {
        "type" : "security_exception",
        "reason" : "no permissions for [indices:admin/shards/search_shards, indices:data/read/search] and User [name=booksuser, backend_roles=[], requestedTenant=__user__]"
      }
    ],
    "type" : "security_exception",
    "reason" : "no permissions for [indices:admin/shards/search_shards, indices:data/read/search] and User [name=booksuser, backend_roles=[], requestedTenant=__user__]"
  },
  "status" : 403
}
```

## Is there anything else you'd like to add?

**Comment by cwillum** [comment]
@gbbafna I started looking at this today and I think I see what’s going on. I’ll make adjustments to the documentation tomorrow for this. If I run into any problems I’ll send a question or two your way. But, as I said, I think I see what this involves. Thanks.

**Comment by cwillum** [comment]
Generated a PR to fix information in documentation: #1053

**Comment by cwillum** [comment]
Closing this issue after fixing in #1053
